website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -6972,6 +6972,40 @@ Timer output timestamps is not yet supported in Python SDK. See https://github.c
 {{< code_sample "sdks/go/examples/snippets/04transforms.go" timer_output_timestamps_good >}}
 {{< /highlight >}}
 
+
+#### 11.3.5 Timer Callback Parameters {#timer-callback-parameters}
+The following parameters are provided for the timer callback methods which could be used for debuging.
+
+1. Window: This can provide the window object to access the window start and end time.
+2. Timestamp: This can provide the timestamp of the element.
```

Comment:
A timer is not necessarily associated with a single element, I think this is more accurate:

```suggestion
2. Timestamp: This can provide the timestamp at which the timer was set to fire.
```

Reply:
lol, I have added this functionality/parameter almost 6 years ago and forgot its details. I should have updated the documentation then.